common-rendering/src/components/liveBlockContainer.tsx

```diff
@@ -29,6 +29,7 @@ const LiveBlockContainer = ({
 					padding: ${space[2]}px ${SIDE_MARGIN}px;
 					padding-left: ${LEFT_MARGIN_DESKTOP}px;
 				}
+				border-bottom: inset;
```

Comment:
Hey @nitro-marky I'm not aware of `inset` as a value for border-bottom. How does this work?

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/CSS/border-style
`inset`
Displays a border that makes the element appear embedded. It is the opposite of outset. When applied to a table cell with border-collapse set to collapsed, this value behaves like groove.

Reply:
I personally think this looks better, like it is embedded on the page but I can change it to a solid line if you want.

Reply:
Interesting! TIL... personally I think it's better to stick with `border-bottom: 1px solid ${neutral[86]};` (or whatever the colour of the line should be. I think this gives us more fine grained control over the border so we can match the design.